0.5.0 (#123)
italia/eidas-it-wallet-docs: docs/en/index.rst

The current Italian scenario counts 3 coexisting digital identity tools that are partially overlapping, sometimes competing, and based on different technologies. This points to a highly fragmented system which yields difficulties for users and service providers. Therefore, the IT Wallet proposes to rationalise the digital identity ecosystem in Italy in order to simplify the experience of citizens, public administrations, and businesses in accessing digital services.

To achieve these objectives and enhance the already active and eIDAS-notified digital identity schemes, the IT Wallet project entails a technological and governance evolution that envisages the progressive migration of the current threefold digital identification solution towards IT Wallet.

The purpose of the following technical rules is to define the technical architecture and reference framework to be used as a guideline by all the parties involved in the development of the IT Wallet project.

> This documentation defines the national implementation profile of EUDI Wallet, containing the technical details about components of the Wallet ecosystem, as listed below:

Author: This document provides technical guidelines for the secure implementation of Wallet ecosystem components that include:

> This documentation defines the national implementation profile of EUDI Wallet, containing the technical details about components of the Wallet ecosystem, as listed below:

Author: ```suggestion
This document provides technical guidelines for the secure implementation of Wallet ecosystem components that include:
```

 - Entities of the ecosystem according to `EIDAS-ARF`_.
 - Infrastructure of trust attesting realiability and eligibility of the participants.

>  - PID and EAAs data schemes and attribute sets.
>  - PID/EAA in MDL CBOR format.
>  - PID/EAA in `SD-JWT`_ format.

Reviewer: ```suggestion
 - PID/(Q)EAAs data schemes and attribute sets.
 - PID/(Q)EAA in MDL CBOR format.
 - PID/(Q)EAA in `SD-JWT`_ format.
```

 - Wallet Solution general architecture.
 - Wallet Instance Attestation.

>  - Issuance of PID/EAA according to `OpenID4VCI`_.
>  - Presentation of PID/EAA according to `OpenID4VP`_.

Reviewer: ```suggestion
 - Issuance of PID/(Q)EAA according to `OpenID4VCI`_.
 - Presentation of PID/(Q)EAA according to `OpenID4VP`_.
```

The current Italian scenario counts 3 coexisting digital identity tools that are partially overlapping, sometimes competing, and based on different technologies. This points to a highly fragmented system which yields difficulties for users and service providers. Therefore, the IT Wallet proposes to rationalise the digital identity ecosystem in Italy in order to simplify the experience of citizens, public administrations, and businesses in accessing digital services.

To achieve these objectives and enhance the already active and eIDAS-notified digital identity schemes, the IT Wallet project entails a technological and governance evolution that envisages the progressive migration of the current threefold digital identification solution towards IT Wallet.

The purpose of the following technical rules is to define the technical architecture and reference framework to be used as a guideline by all the parties involved in the development of the IT Wallet project.

> In this documentation there are the technical details for implementing the components that compose the Wallet ecosystem:

Reviewer 2: [Minor recommendation]:
`This document provides technical guidelines for the secure implementation of Wallet ecosystem components that include:'

Author: Done
